packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -297,6 +297,7 @@ - (void)loadApp:(NSURL *)expoUrl onSuccess:(void (^ _Nullable)(void))onSuccess o
   expoUrl = [EXDevLauncherURLHelper replaceEXPScheme:expoUrl to:@"http"];
 
   NSDictionary *updatesConfiguration = [EXDevLauncherUpdatesHelper createUpdatesConfigurationWithURL:expoUrl
+                                                                                          projectURL:expoUrl
```

Comment:
from what I understand, the project url is the `EXUpdatesURL` ? should we get that url from the `Expo.plist` here?